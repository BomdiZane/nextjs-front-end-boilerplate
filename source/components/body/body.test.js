import Body from './body';

describe('Body component:', () => {
	const body = shallow(<Body />);

	it('should match snapshot', () => expect(body).toMatchSnapshot());
});
