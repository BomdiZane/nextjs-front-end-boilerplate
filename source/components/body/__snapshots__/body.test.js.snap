// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Body component: should match snapshot 1`] = `
<Body
  classes={
    Object {
      "content": "Body-content-1",
    }
  }
/>
`;
